cli/cmd/generate_support_archive.go

```diff
@@ -156,6 +195,75 @@ keptn generate support-archive --dir=/some/directory`,
 	},
 }
 
+func isConfiguredIngressGatewayAvailable() *errorableBoolResult {
+	res, err := exechelper.ExecuteCommand("kubectl", "get cm -n keptn ingress-config -ojsonpath='{.data.ingress_gateway}'")
+	if err != nil {
+		return newErrorableBoolResult(false, err)
+	}
+	var gatewayFullName string
+	if res == "" {
+		gatewayFullName = "public-gateway.istio-system"
+	} else {
+		gatewayFullName = res
+	}
+
+	var gatewayName, gatewayNamespace string
+	gwSplit := strings.Split(gatewayFullName, ".")
+
+	gatewayName = gwSplit[0]
+	if len(gwSplit) > 1 {
+		gatewayNamespace = gwSplit[1]
+	} else {
+		gatewayNamespace = "istio-system"
+	}
+
+	res, err = exechelper.ExecuteCommand("kubectl", "get gateway -n "+gatewayNamespace+" "+gatewayName)
```

Comment:
Can we here please add `-oyaml` or `-ojson` in order to get all details of the gateway?

Reply:
Detailed information about all available gateways is provided in the output generated by 	https://github.com/keptn/keptn/blob/dc66510648c7f73071aff7a4630978c525677855/cli/cmd/generate_support_archive.go#L228

Reply:
Thanks!